enhancements/kube-apiserver/audit-policy.md

```diff
@@ -287,6 +311,11 @@ The profile translate like this:
         # catch-all rule to log all other requests with request and response payloads
         - level: RequestResponse
     ```
+  
+With secure OAuth storage in-place (compare "Logging of Token with Secure OAuth Storage" section) and
```

Comment:
compare "Logging of Token with Secure OAuth Storage" section
should be
`compared to`

Reply:
it should be a reference. I write "see" instead.